### PR TITLE
Adding to electrical/software sections

### PR DESCRIPTION
**Pull request by xcarcelle**

### CODE REVIEW

**Comment by xcarcelle** [comment]
Hi @ronkorving : do you have any comments on my last PR ?
Regards.

**Comment by xcarcelle** [comment]
Dear @ronkorving : any comments ? I will push more words if fine with you.

**Comment by xcarcelle** [comment]
Did push some more words to this PR

**Comment by xcarcelle** [comment]
Hi guys,
are you still interested by this commits ? 
Can I push more commit when you validate those ones ?
Cheers.

**Comment by ronkorving** [comment]
Hi @xcarcelle, still interested! Sometimes things fall on the back burner.
I'm reviewing your PR now. May take a little while to check each word (I'm confirming with native Japanese speakers). Feel free to push more.

File: ReadMe.md [hunk 1]
```diff
@@ -71,8 +71,10 @@ To reboot, restart | 再起動（する） | さいきどう |
 To update | 更新（する） | こうしん |
 To register | 登録（する） | とうろく |
 To add | 追加（する） | ついか |
+To insert/include | 組み込み (する) | くみこみ |
 To edit | 編集（する） | へんしゅう |
 To delete / remove | 削除（する） | さくじょ |
+Processing | 処理中 | しょりちゅう |
```
Comment (ronkorving) [inline ReadMe.md]: Perhaps just `処理`?
You can of course use `処理中` for cases where "it is currently processing", but just the word for the concept of processing something, is `処理`, right?

File: ReadMe.md [hunk 2]
```diff
@@ -144,7 +146,9 @@ Type | 型 | かた | As in data type
 Variable | 変数 | へんすう |
 Variable name | 変数名 | へんすうめい |
 Value | 値 | あたい |
+Initial value | 初期値 | しょきち |
 Null | ヌル | |
+Concatenation | 定数 | ていすう |
```
Comment (ronkorving) [inline ReadMe.md]: I think `定数` means constant?

File: ReadMe.md [hunk 3]
```diff
@@ -278,10 +284,14 @@ Conduction | 導電 | どうでん |
 Conductivity | 導電性 | どうでんせい |
 Connector | コネクタ | |
 Current | 電流 | でんりゅう |
+Daisy-chain | 橋渡し | はしわたし |
```
Comment (ronkorving) [inline ReadMe.md]: I can't find examples of this via Google.
jisho.org mentions a different Japanese expression though: https://jisho.org/search/daisy%20chain

File: ReadMe.md [hunk 4]
```diff
@@ -278,10 +284,14 @@ Conduction | 導電 | どうでん |
...
 Datasheet | データシート | |
+Detection | 検知 | けんち |
 Diode | ダイオード | |
+Display | 表示 | ひょうじ |
```
Comment (ronkorving) [inline ReadMe.md]: This word is already mentioned under `Computer and Application Usage`

File: ReadMe.md [hunk 5]
```diff
@@ -290,14 +300,19 @@ Integrated Circuit (IC) | 集積回路 | しゅうせきかいろ
 Layer | 層 | そう | E.g. 4-layer ⇒ 4層
 Layout | パターン設計 | パターンせっけい | As in "PCB layout"
 Layout drawing | パターン図 | パターンず | As in "PCB layout"
-To Measure | 測定（する） | そくてい
+Magnet | 磁石 | じしゃく |
+To Measure | 測定（する） | そくてい |
```
Comment (ronkorving) [inline ReadMe.md:304]: "Measurement"?

File: ReadMe.md [hunk 6]
```diff
@@ -290,14 +300,19 @@ Integrated Circuit (IC) | 集積回路 | しゅうせきかいろ
...
 Microcontroller (MCU) | マイコン | |
+Modulator | 変調 | へんちょう |
```
Comment (ronkorving) [inline ReadMe.md]: I expect this is "Modulation", where a modulator would be a `変調器`?

File: ReadMe.md (hunk 1 above)
Comment (xcarcelle) [reply ReadMe.md]: right

File: ReadMe.md (hunk 2 above)
Comment (xcarcelle) [reply ReadMe.md]: 連結 might be more accurate I believe

File: ReadMe.md (hunk 3 above)
Comment (xcarcelle) [reply ReadMe.md]: It is used by japanese colleagues when talking about electronic devices daisy-chaining. Could you check with other EE japanese engineers to double-check ?

File: ReadMe.md (hunk 4 above)
Comment (xcarcelle) [reply ReadMe.md]: removing

File: ReadMe.md (hunk 6 above)
Comment (xcarcelle) [reply ReadMe.md]: right

File: ReadMe.md (hunk 3 above)
Comment (ronkorving) [reply ReadMe.md]: If I knew any, I would :)

File: ReadMe.md (hunk 2 above)
Comment (ronkorving) [reply ReadMe.md]: And I guess it's a suru-verb?

Comment (xcarcelle) [reply ReadMe.md]: changed to suru-verb

File: ReadMe.md (hunk 3 above)
Comment (xcarcelle) [reply ReadMe.md]: removed from now, will look for more examples

**Comment by xcarcelle** [comment]
@ronkorving : last commit fixed your comments ?

**Comment by ronkorving** [comment]
Cheers! Merged!

**Pull request merged**